nrf_802154/doc/CHANGELOG.rst

```diff
@@ -280,7 +284,7 @@ Added
 Notable Changes
 ===============
 
-* The release notes of the legacy versions of the Radio Driver are available in the Changelog for 802.15.4 Radio Driver v1.10.0.
+* The release notes of the legacy versions of the Radio Driver are available in the `Radio Driver section`_ of the Infocenter.
```

Comment:
@piotrkoziar , please bcakport the content of CHANGELOG.rst to internal repo.
For now we could proceed with this deploy provided that we do not change this line of changelog.

Reply:
Removed the change.